community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/NodeUpdates.java

```diff
@@ -48,6 +47,7 @@
 public class NodeUpdates implements PropertyLoader.PropertyLoadSink
 {
     private final long nodeId;
+    private static final long[] EMPTY_LONG_ARRAY = new long[0];
```

Comment:
Just a note: there's a `PrimitiveLongCollections#EMPTY_LONG_ARRAY` too, but I won't request change in this PR.